map/bookmark_helpers.cpp

```diff
@@ -345,23 +345,24 @@ std::unique_ptr<kml::FileData> LoadKmlFile(std::string const & file, KmlFileType
   return kmlData;
 }
 
-std::string GetKMLorGPXPath(std::string const & filePath)
+std::vector<std::string> GetPathsToLoad(std::string const & filePath)
 {
   std::string const fileExt = GetLowercaseFileExt(filePath);
-  std::string fileSavePath;
   if (fileExt == kKmlExtension)
   {
     // Copy input file to temp output KML with unique name.
-    fileSavePath = GenerateValidAndUniqueFilePathForKML(base::FileNameFromFullPath(filePath));
+    std::string fileSavePath = GenerateValidAndUniqueFilePathForKML(base::FileNameFromFullPath(filePath));
     if (!base::CopyFileX(filePath, fileSavePath))
       return {};
+    return {fileSavePath};
   }
   else if (fileExt == kGpxExtension)
   {
     // Copy input file to temp GPX with unique name.
-    fileSavePath = GenerateValidAndUniqueFilePathForGPX(base::FileNameFromFullPath(filePath));
+    std::string fileSavePath = GenerateValidAndUniqueFilePathForGPX(base::FileNameFromFullPath(filePath));
```

Comment:
ditto

Reply:
Replaced with 'auto fileSavePath' as suggested for other cases